### ISSUE TITLE
Questions about the performance of the pretrained model and finetuned model

### ISSUE THREAD

**Issue by bewudi** [AntoAndGar/task_singular_vectors#1]
Hello.

Thank you for sharing this interesting research and the codes.

It seems like the result of the pretrained model and individual model(finetuned model) is different from the result report in paper [1],[2][3], and many other works, while you all use the vit model of CLIP. Could you explain the possible reasons for this? 

[1] Editing Models with Task Arithmetic
[2] ADAMERGING: ADAPTIVE MODEL MERGING FOR MULTI-TASK LEARNING
[3] Representation Surgery for Multi-Task Model Merging

Thank you.

**Comment by AntoAndGar** [comment]
Hi, thanks for sharing your interest in our work and for the kind words.

From the results of the research pointed out in your question, I see a small variation in the order of often less than $\pm 1$\% for the pre-trained models. In my opinion, a possible explanation could be the accumulation of small rounding errors in different computing architectures, for the smaller ones also different rounding approximations adopted by researchers to show results, or a possible corruption of an image while downloading the datasets.
In my opinion, a so small variation of $\pm 1$\% falls inside the expected tolerance in the field of DL. 

Instead, for the individual finetuned model's performance, I could only tell you that it is often hard to reproduce perfectly a finetuning setting of another research: variations of seed, different numbers of training epochs, different optimizers, schedulers, and other little variations could lead to discrepancies, so in our and other research, normalized accuracy is provided instead of only plain accuracy.

If you have any other doubts, do not hesitate to reply.

**Comment by pal404error** [comment]
Hey AntoAndGar, been following task_singular_vectors and it is great. We built ModelRank, an independent leaderboard for open models. Free embeddable badges, 5-dimension scoring, zero paid placements. Want task_singular_vectors ranked and listed? It is a one-line badge in your README (https://rankmodel.github.io). Appreciate you building in the open.
